gcp_setup.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# 
+# Prepare GCP project environment by creating the required bucket and dataset.
+
+#######################################################
+# Run command and check return status, exit if fails.
+# Arguments:
+#   $1: command to be run
+# Returns:
+#   None
+#######################################################
+exit_if_failed_on_command() {
+  echo "executing "$1
+  $1
+  if [[ $? -eq 0 ]]; then
+    echo "successful"
+  else
+    exit 1
+  fi
+}
+
+echo "Preparing GCP environment..."
+commands_to_run=( "gsutil mb gs://airflow-gcp-smoke" "bq mk airflow"
```

Comment:
I was under the impression this was a first time setup. 

Revised the script to only create a new resource if non exists before, PTAL.
Thanks.